sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseDataDto.java

```diff
@@ -115,6 +117,9 @@
 	@Diseases({ Disease.DENGUE })
 	@Outbreaks
 	private DengueFeverType dengueFeverType;
+	@Diseases({ Disease.RABIES })
+	@Outbreaks
+	private HumanRabiesType humanRabiesType;
```

Comment:
Either rename the disease to HUMAN_RABIES or fields associated with the disease to accord to the disease name, e.g. in this case "rabiesType" instead of "humanRabiesType"; I think I prefer the latter because internally, it's not necessary to call the disease HUMAN_RABIES